andela/travel_tool_front
# Add test cases tests for the Travel Readiness Component

#### What does this PR do?
- Improve test coverage for component to 100%
#### Description of Task to be completed?
-Add test cases for the Travel Readiness Component

#### How should this be manually tested?

- git clone this repo `https://github.com/andela/travel_tool_front.git`
- git checkout this branch `increase-tests-coverage`
- run yarn install
- run `yarn test`. All tests should pass

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[161724530](https://www.pivotaltracker.com/story/show/161724530)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
